### PR TITLE
[ADD] stock aggregated shipment document webkit

### PR DESCRIPTION
**Pull request by max3903**

### CODE REVIEW

**Comment by lepistone** [comment]
Hi @max3903, the commits by your colleague Marc Cassuto are not attributed to any github user. He might have another email address, or not have an account at all. Is that intentional? Thanks!

**Comment by max3903** [comment]
@lepistone Marc is no longer working for Savoir-faire Linux.

**Comment by lepistone** [comment]
@max3903 ah, I see, no problem.

File: stock_aggregated_shipment_documents_webkit/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+###############################################################################
+
+
+{
+    'name': 'Aggregated Bill oif Lading & Commercial Invoice',
```
Comment (yvaucher) [inline stock_aggregated_shipment_documents_webkit/__openerp__.py:25]: s/oif/of

File: stock_aggregated_shipment_documents_webkit/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    'version': '1.0',
+    'category': 'Reports/Webkit',
+    'description': """
+Aggregated Bill of Lading & Commercial Invoice - webkit report
+==============================================================
+
+For international shipments, carriers have to present a Bill of Lading and a
+Commercial Invoice to the customs.  When shipping multiple orders to the
+same customer or to an intermediary Warehouse, you need to produce a unique
+Bill of Ladding which aggregate all the selected Delivery Orders and a
```
Comment (yvaucher) [inline stock_aggregated_shipment_documents_webkit/__openerp__.py:35]: s/aggregate/aggregates/

**Comment by mcassuto** [comment]
@lepistone : @max3903 is right, I am not at Savoir-Faire anymore... I sent my OCA CLA with my new email address and github login, I don't know why they did made the change...

**Comment by lepistone** [comment]
Ok @mcassuto.

I don't think there was any change: these are old commits you made when your email was at SFL, and now that the email associated to your github account has changed, the commits are not associated to any github account anymore.

One possible solution is to add as a secondary, private address the old SFL email address to github. This way your old commits are recognized by github as associated to your accounts.

What do you think?

**Pull request closed without merging**